docs/snippets/angular/component-story-custom-args-complex.ts.mdx

```diff
@@ -0,0 +1,46 @@
+```ts
+// YourComponent.stories.ts
+
+import { Story, Meta } from '@storybook/angular/types-6-0';
+import YourComponent from './your.component';
+
+// a function to apply some computations
+const someFunction = (valuePropertyA: string, valuePropertyB: string) => {
+  // makes some computations and returns something
```

Comment:
Is it common in the snippets to use comments instead of some sort of dummy example? Sorry, still learning the ropes.

Reply:
@phated no need to apologize whatsoever. We're all learning here. In terms of your comment i went with a generic approach as this could be anything the reader wants to be and also keeps the snippet shorter